src/main/java/com/stripe/model/Subscription.java

```diff
@@ -908,7 +908,7 @@ public static class PaymentMethodOptions extends StripeObject {
      * invoices created by the subscription.
      */
     @SerializedName("customer_balance")
-    CustomerBalance customerBalance;
```

Comment:
TODO

Reply:
@kamil-stripe was working on this and should be able to help here.

Reply:
In the end, it was @richardm-stripe that figured it out. Took us a good while though.